op_builder/cpu/comm.py

```diff
@@ -42,3 +42,30 @@ def extra_ldflags(self):
             return []
         else:
             return ['-lccl', f'-L{ccl_root_path}/lib']
+
+
+class SHMCommBuilder(CPUOpBuilder):
```

Comment:
@delock, please feel free to disregard this optional feedback. I think it would be more readable to rename `SHM` to something like `SharedMem`. The first thing that comes to mind when reading the code is `shim` :) https://stackoverflow.com/questions/2116142/what-is-a-shim

I am probably the only person affected like this, so please feel free to disregard.

Reply:
Hi @tjruwase , thanks for the comment.   I'll change the class name to make it more clear.